lib/semian/sysv_integer.rb

```diff
@@ -0,0 +1,12 @@
+module Semian
+  module SysV
+    class Integer < Semian::Simple::Integer #:nodoc:
```

Comment:
Yeah, I meant with inheritance from C, but that probably doesn't give you anything and gives more poor visibility.